data/release/demo/WCS_getCoverage_2.0.1_scale_factor.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Demonstrates use of WCS scale factor -->
+<wcs:GetCoverage service="WCS" version="2.0.1"
+    xmlns:wcs="http://www.opengis.net/wcs/2.0" xmlns:scal="http://www.opengis.net/WCS_service-extension_scaling/1.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.opengis.net/wcs/2.0 http://schemas.opengis.net/wcs/2.0/wcsAll.xsd">
+    <!--
```

Comment:
Hmm... will we remember to remove the comment once the other issue is fixed? Does it actually need to be here?